libvips/foreign/svgload.c

```diff
@@ -315,6 +325,9 @@ vips_foreign_load_svg_file_header( VipsForeignLoad *load )
 
 static const char *vips_foreign_svg_suffs[] = {
 	".svg",
+#if LIBRSVG_CHECK_FEATURE(SVGZ)
```

Comment:
No, it'll work without zlib, librsvg uses libgsf internally and matching on file extension does not go through the buffer detection code (unless I'm confused ;-).